frontend/cypress/integration/talentsearch/search-workflows.spec.js

```diff
@@ -25,6 +25,8 @@ describe("Talent Search Workflow Tests", () => {
     // second request is properly filtered
     searchReturnsGreaterThanZeroApplicants();
 
+    cy.findByRole("combobox", { name: /Classification/i }).click().wait(5000).type('{enter}');
```

Comment:
Do we need the `.wait(5000)` here?

Reply:
when running it locally, there was sometimes a small lag before the multiselect loaded options when cypress clicked on it
this sometimes resulted in a failure to select anything

this might have just been a local run quirk, I've taken Peter's suggestion and so far it is testing fine here